<launch>	
	<master auto="start"/>
	<node pkg="prolog_interface" type="prolog_interface" name="prolog_interface" output="screen">
		<param name="prolog_binary" value="/usr/lib/swi-prolog/bin/amd64" type="str"/> 
		<param name="prolog_path" value="/home/guglielmo/ros_workspace/groovy/SapienzBotReasoning/pl" type="str"/>  
	</node> 
</launch>
